src/esp/assets/ResourceManager.h

```diff
@@ -366,86 +359,99 @@ class ResourceManager {
     return collisionMeshGroups_.at(collisionAssetHandle);
   }
 
-  /**
-   * @brief Get the index in @ref physicsObjTemplateLibrary_ for the object
-   * template asset identified by the key, templateHandle.
-   *
-   * @param templateHandle The key referencing the asset in @ref
-   * physicsObjTemplateLibrary_.
-   * @return The index of the object template in @ref
-   * physicsObjTemplateLibrary_.
-   */
-  int getObjectTemplateID(const std::string& templateHandle) const {
-    const bool objTemplateExists =
-        physicsObjTemplateLibrary_.count(templateHandle) > 0;
-    if (objTemplateExists) {
-      return physicsObjTemplateLibrary_.at(templateHandle)
-          ->getObjectTemplateID();
-    }
-    return ID_UNDEFINED;
-  }
-
   /**
    * @brief Get the key in @ref physicsObjTemplateLibrary_ for the object
    * template asset index.
    *
    * @param objectTemplateID The index of the object template in @ref
    * physicsObjTemplateLibrary_.
-   * @return The key referencing the asset in @ref physicsObjTemplateLibrary_.
+   * @return The key referencing the asset in @ref physicsObjTemplateLibrary_,
+   * or an empty string if does not exist.
    */
-  std::string getObjectTemplateHandle(const int objectTemplateID) const;
+  std::string getObjectTemplateHandle(const int objectTemplateID) const {
+    CORRADE_ASSERT(physicsTemplatesLibByID_.count(objectTemplateID) > 0,
+                   "ResourceManager::getObjectTemplateHandle : No loaded or "
+                   "primitive template with index "
+                       << objectTemplateID << " exists.",
```

Comment:
```suggestion
    CORRADE_ASSERT(physicsTemplatesLibByID_.count(objectTemplateID) > 0,
                   "ResourceManager::getObjectTemplateHandle : No loaded or "
                   "primitive template with index"
                       << objectTemplateID << "exists.",
```

FYI, Corrade's Debug (and also all Corrade assert macros) implicitly add a space between the values, so adding one more would cause double spacing. Same below.